polaris-react/src/components/Checkbox/Checkbox.scss

```diff
@@ -18,7 +18,7 @@
       #{$se23} & {
         outline: var(--p-border-width-2) solid
           var(--p-color-border-interactive-focus);
-        outline-offset: var(--p-space-05);
+        outline-offset: var(--p-space-025);
```

Comment:
|Before|After|
|-|-|
|<img width="144" alt="Screenshot 2023-06-09 at 3 17 00 PM" src="https://github.com/Shopify/polaris/assets/20652326/f0059d12-7fa3-4325-a902-765d5ea9b0d4">|<img width="150" alt="Screenshot 2023-06-09 at 3 17 21 PM" src="https://github.com/Shopify/polaris/assets/20652326/c3f9b4fe-2523-4385-989c-333c5a34b314">|

Reply:
I changed this to 025 yesterday but @gwyneplaine changed it back overnight. Was causing a legit issue https://github.com/Shopify/polaris/pull/9426#event-9485668849

Reply:
the size issue? its still working with the `025` on this PR! supposed to be `16px`x`16px` on `md-up` and `18px`x`18px` on small screens